tests/core/basic20/task.yaml

```diff
@@ -58,4 +61,11 @@ execute: |
     done
 
     # ensure the "snap recovery" command works
-    snap recovery | MATCH '[0-9]+\ +canonical\*\ +ubuntu-core-20-[^ ]+\ +current'
+    MODEL="$(snap model | grep '^model' | awk '{ print $2 }')"
+    BRAND="$(snap model | grep '^brand' | cut -d' ' -f2- | xargs)"
+    if echo "$BRAND" | MATCH ".*\(.*\)"; then
```

Comment:
then this logic can go away too if you use `--verbose`

Reply:
thanks

Reply:
sorry if I was unclear, but I mean this whole if statement can go away since you are using --verbose, since with --verbose, we don't combine the display-name and model name, so you can always use `$MODEL` as-is from the above grep and awk command